pubspec.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
   source_maps: "^0.10.5"
   source_span: "^1.6.0"
   stack_trace: ">=0.9.0 <2.0.0"
-  stream_transform: ">=0.0.20 <2.0.0"
+  stream_transform: ">=0.0.20 <3.0.0"
```

Comment:
We just published the stable release of `stream_transform`. It'll be picked up by a fresh `pub get`.

@nex3 – if you restart the tests here, we can confirm it's picked up on the Dart `dev` builds to ensure nothing is broken.

Reply:
I rebased the branch on top of master so the tests got restared as well. The new stable `stream_transform` was picked and it seems that everything works fine.

Reply:
@nex3 ?